.github/scripts/spellcheck.words

```diff
@@ -20,6 +20,8 @@ AmigaOS
 AmiSSL
 anyauth
 anycast
+apache
+Apache
```

Comment:
We do have a chance here to insist on correct casing by not accepting `apache` as correct. Just saying, I have no strong opinion.

Reply:
Both are used frequently elsewhere. I think both are fine.